templates/frontend/pages/search.tpl

```diff
@@ -90,7 +90,7 @@
 	{* Search results, finally! *}
 	<div class="search_results">
 		{iterate from=results item=result}
-			{include file="frontend/objects/article_summary.tpl" article=$result.publishedSubmission journal=$result.journal showDatePublished=true hideGalleys=true}
+			{include file="frontend/objects/article_summary.tpl" heading="h3" article=$result.publishedSubmission journal=$result.journal showDatePublished=true hideGalleys=true}
```

Comment:
Is there a `<h2>` element on the `search.tpl` page? I did a quick scan and didn't see one.

Reply:
Did you have a chance to investigate this comment?

Reply:
Nope, didn't find any `<h2>`s!

Reply:
Fixed this in https://github.com/pkp/ojs/pull/2692/commits/34d5d51fb90e9b4164970c0c46c3193e2b897625